examples/ex9.cpp

```diff
@@ -191,6 +201,9 @@ int main(int argc, char *argv[])
    }
    args.PrintOptions(cout);
 
+   Device device(device_config);
+   device.Print();
+
```

Comment:
It is really awesome to see the Device support spread through the examples! (So far in ex1, ex1p, ex3, ex3p, ex6, ex6p, ex9, ex9p, ex24 and ex24p)